cpp/src/arrow/array/array_list_test.cc

```diff
@@ -233,6 +233,26 @@ class TestListArray : public ::testing::Test {
                                                  expected->null_bitmap()));
   }
 
+  void TestFromArraysWithSlicedNullBitmap() {
```

Comment:
For the record, do we already have a test with sliced offsets but no validity bitmap?

Reply:
I don't think so. I will add one.